wlroots/wlr_types/box.py

```diff
@@ -16,22 +30,7 @@ def __init__(self, x: int, y: int, width: int, height: int) -> None:
         self._ptr.width = width
```

Comment:
I'm not sure why I had separated out `self._x` and `self._ptr.x`, but that may be something we can collapse.

Reply:
:+1: